Havit.Blazor.Components.Web.Bootstrap/Tags/Internal/HxInputTagsInternal.razor.css

```diff
@@ -17,7 +17,7 @@ input:focus {
 }
 
 ::deep .hx-tag {
-	margin: var(--hx-input-tags-tag-margin);
+	margin: var(--hx-input-tags-tag-gap);
```

Comment:
@rqx110 the property name should be `gap`.
`gap: var(--hx-input-tags-tag-gap);`

Reply:
the `gap` property can't work for increase gap(vertical) between tags!
![image](https://github.com/user-attachments/assets/aa7d70f9-7383-4ed2-a624-4def4f12edc2)

Reply:
Yes, because it needs to be set on the container, not the tag itself.

Reply:
So as i said before...

> lets do it with a gap on **hx-input-tags-control** class.

Reply:
got it!